docs/source/daml-repl/index.rst

```diff
@@ -60,11 +60,16 @@ two forms:
 1. An expression ``expr`` of type ``Script a`` for some type ``a``. This
    will execute the script ignoring the result.
 
-2. A binding of the form ``x <- expr`` where ``x`` is a variable name
-   and ``expr`` is an expression of type ``Script a``. This will
-   execute the script and bind the result to the variable ``x``. You
+2. A binding of the form ``pat <- expr`` where ``pat`` is pattern, e.g.,
+   a variable name to bind the result to
+   and ``expr`` is an expression of type ``Script a``.
+   This will execute the script and bind the result to the variable ``x``. You
```

Comment:
`x` is no longer defined in the beginning of the sentence.

Reply:
Good catch, fixed!